acceptance/testdata/launcher/exec.d/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/buildpacks/lifecycle/acceptance/testdata/launcher/exec.d
+
+go 1.17
```

Comment:
I tried many ways of setting `GO111MODULE=off` in the Windows environment and it did not work. I finally gave up and added this file.